### ISSUE TITLE
Add update command

### ISSUE THREAD

**Issue by davsclaus** [minishift/minishift#72]
It would be good with an update command as not many users would know that its the status command that can tell if there is a newer version and do an auto update.

**Comment by jimmidyson** [comment]
BTW all commands apart from version trigger an update check.

**Comment by davidaparicio** [comment]
```
There is a newer version of minishift available. Do you want to
automatically update from v0.8.0 to v0.9.0 now? [y/N] Y
Skipping autoupdate
```
It asks to me, only one time..
Why "Y" isn't working? Is it only "y" (lowercase)?

**Comment by LalatenduMohanty** [comment]
Yes, it seems it is only checking for a small `y` 

```
                if confirm == "y" {
                        fmt.Printf("Updating to version %s\n", latestVersion)
                        updateBinary(latestVersion, downloadBinary, updateLinkPrefix, downloadLinkFormat)
                        return
                }
```

**Comment by jimmidyson** [comment]
@davidaparicio @LalatenduMohanty Fixed in #127

**Comment by hferentschik** [comment]
I kind of like the idea of an explicit _upate-vm_ command. There is the auto-update, but at least imo it gets into the way every now and then and people also report that they error/warning messages from exceeding the rate limit towards the GitHub API. I think I'd prefer a dedicate command to check the VM version and potentially update it. 

Also, given there is not much feature wise which changes in the VM over time, I wonder whether the automatic checks are not a bit of an overkill anyways.

**Comment by hferentschik** [comment]
Merged via pull request #930
